python/CMakeLists.txt

```diff
@@ -1,15 +1,39 @@
-if(WIN32 AND CMAKE_BUILD_TYPE STREQUAL "Debug")
-  # pybind11 logic for setting up a debug build when both a debug and release
-  # python interpreter are present in the system seems to be pretty much broken.
-  # This works around the issue.
-  set(PYTHON_LIBRARIES "${PYTHON_DEBUG_LIBRARIES}")
+# Detect if we are doing a standalone build of the bindings, using an external sdformat
+if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
+  cmake_minimum_required(VERSION 3.22.1)
```

Comment:
Can you document this CMake version requirement?

Reply:
https://github.com/gazebosim/sdformat/pull/1497/commits/ea6256a4ea4cfd90b90572d321a078869188eaaa

thanks for catching these!